tasks/master.yml

```diff
@@ -19,7 +19,7 @@
 - name: Add trust in pg_hba.conf
   lineinfile:
     state: present
-    dest: "{{ pgsqlrep_data_path }}/pg_hba.conf"
+    dest: "{{ pgsqlrep_config_path }}/pg_hba.conf"
```

Comment:
Same here. I'm not sure what the value of `pgsqlrep_config_path` is. The path to this file on RHEL7 is `/var/lib/pgsql/9.6/data/pg_hba.conf`.